views/marts/product/fct_feature_daily_snapshot.view.lkml

```diff
@@ -0,0 +1,228 @@
+view: fct_feature_daily_snapshot {
+  sql_table_name: "MART_PRODUCT"."FCT_FEATURE_DAILY_SNAPSHOT" ;;
+
+
+  ## Keys
+
+  dimension: daily_server_id {
+    type: string
+    sql: ${TABLE}."DAILY_SERVER_ID" ;;
```

Comment:
nit: the column names have to be lower case, to be consistent with other views in the folder. 

`${TABLE}."DAILY_SERVER_ID"` to `${TABLE}.daily_server_id`

Reply:
I did it for now, but as discussed in our meeting, let's decide if there's value in following the old practice.